integration/kubernetes/init.sh

```diff
@@ -155,6 +155,8 @@ echo "Kubelet options:"
 sudo cat /var/lib/kubelet/kubeadm-flags.env
 sudo systemctl daemon-reload && sudo systemctl restart kubelet
 
+sleep 5
```

Comment:
why we need this sleep?

Reply:
wip, just want to see if it helps to avoid the `Unable to connect to the server: net/http: TLS handshake timeout`

Reply:
@GabyCT this error seems to be pretty common, mostly in single node clusters AFAICT, i couldn't find any solution for this.
In our case the sleep seems to help, i tried several runs and didn't get the error, hence, I suggested simple backoff sleep in the recent push